src/App.js

```diff
@@ -1,58 +1,70 @@
-import React, { useState, useEffect } from 'react';
-import './App.css';
+import "./App.css";
+import "normalize.css";
```

Comment:
jesli uzywasz takiego czarodzieja, to przesuń go do src/ :)

Reply:
Czarodziej poleciał do kosza (łapałam się wszystkiego :D )